includes/datatypes.php

```diff
@@ -400,7 +400,7 @@ function add_podcasting_term_meta_nonce( $term, $taxonomy = false ) {
 
 	if ( $taxonomy ) {
 		$url = get_term_feed_link( $term->term_id, TAXONOMY_NAME );
-		__( 'Your Podcast Feed:', 'ads-txt' );
+		__( 'Your Podcast Feed:', 'simple-podcasting' );
```

Comment:
Does this actually output? Maybe I'm missing something here but I'm not actually seeing an echo statement around this so seems like this won't ever display. If so, this should probably be `esc_html_e`

Reply:
@dkotter thank you! This is not related to translations, I created a new issue #133